README.md

```diff
@@ -20,6 +20,13 @@ omnect-cli is a cli tool to manage your omnect-devices. It provides commands to
 - SSH
   - Open an ssh tunnel on a device in the field to connect to it.
 
+# Prerequisites
+
+Depending on your intended use use you want to install the following packages.
+
+- openssh: For the `ssh` command.
+- docker: For the `file`, `iot-hub-device-update`, `identity` and `wifi` commands.
```

Comment:
okay. 
currently the install description implicitly has docker as a prerequisite.